README.md

```diff
@@ -201,8 +208,20 @@ Build and test orchestration is performed by [Jenkins][21].
 [17]:   https://www.arm.com/
 [18]:   https://www.mininodes.com/
 [19]:   http://techtribe.nl
+[1]:    irc://irc.freenode.net/node-build
 [20]:   https://tessel.io/
 [21]:   https://jenkins.io/
+[2]:    https://digitalocean.com/
+[3]:    https://www.rackspace.com/
+[4]:    https://www.rackspace.com/cloud/files
+[5]:    https://www.mailgun.com/
+[6]:    https://github.com/nodejs/email
+[7]:    https://www.microsoft.com/
+[8]:    https://azure.microsoft.com
+[9]:    https://www.joyent.com/
```

Comment:
I think I prefer the numbers listed in numerical order.

Reply:
Done